scripts/cmd/_run_enceladus.cmd

```diff
@@ -658,13 +665,25 @@ EXIT /B 0
 EXIT /B 0
 
 :temp_log_file
-    SET yyyy=%date:~-4%
-    SET MM=%date:~3,2%
-    SET DD=%date:~0,2%
-    ::alternate date parsing
-    ::SET YYYY=%date:~10,4%
-    ::SET MM=%date:~4,2%
-    ::SET DD=%date:~7,2%
+    IF %DATE_PARSING_TYPE%==parser1 (
```

Comment:
In the most usual cases, this will probably work fine.
In general, the users can change the formats, so even the position-based parsing may not be sufficient (e.g. for the situation of (non)-padding with 0, short/long year, etc.

I suggest taking inspiration at https://ss64.com/nt/syntax-gettime.html

Reply:
Nice. I will try to implement this in a free moment. This is no high priority. I pushed the change only because I needed a different parsing, so made it switchable, albeit not the smartest. 😉

Reply:
Yes, I agree that the current solution is probably good enough for the use case. 

It just made sense to provide some information in the problematics in the broader sense, we may very well decide to leave it as-is, it would just be a bit more informed decision 😄

Reply:
Should be universal now 🤞 
Also added configuration to prevent editor conversion from CRLF to LF